src/realm/utilities.hpp

```diff
@@ -140,6 +140,18 @@ int fast_popcount32(int32_t x);
 int fast_popcount64(int64_t x);
 uint64_t fastrand(uint64_t max = 0xffffffffffffffffULL, bool is_seed = false);
 
+class FastRand {
+public:
+    FastRand(uint64_t seed = 1)
+        : state(seed)
+    {
+    }
+    uint64_t operator()(uint64_t max = 0xffffffffffffffffULL);
```

Comment:
How about a note, that this is not a thread safe random generator?

Reply:
Added some comment